src/styles/base.css

```
@@ -0,0 +1,798 @@
+  transform: translateY(-4px);
+  box-shadow: 0 10px 25px rgba(0, 0, 0, 0.1);
+  border-color: #3b82f6;
+}
+
+.calculator-link {
+  display: block;
+  padding: 24px;
+  text-decoration: none;
+  color: inherit;
+  height: 100%;
+  display: flex;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)